meson.build

```diff
@@ -330,9 +335,27 @@ murmur3_dep = dependency('murmur3', version: '>=0.1.0', fallback: ['murmur3', 'm
 secp256k1_dep = dependency('libsecp256k1', version: '>=0.1.0', fallback: ['secp256k1', 'secp256k1_dep'])
 softfloat3_dep = dependency('softfloat3', version: '>=3.0.0', fallback: ['softfloat3', 'softfloat3_dep'])
 
+# Our ge additions must be in their own library because they cannot be compiled
+#  with '-funsigned-char' since they rely on signed characters.
```

Comment:
On cc-release, this should probably be a separate derivation in `pkg/ge-additions` or somesuch.

Reply:
Moved to cc-release and put this code in a separate `pkg/ge-additions` package.